pkg/options/openapi/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why do we need this here? We already have it at the root level.

Reply:
Well, I copied this file wholesale from K8S, so I felt like I needed to copy the license. WDYT?

Reply:
Removed.